core/modules/views/plugins/views_plugin_display_block.inc

```diff
@@ -70,20 +70,24 @@ class views_plugin_display_block extends views_plugin_display {
    */
   function execute_hook_block_list($delta = 0, $edit = array()) {
     $delta = $this->view->name . '-' . $this->display->id;
-    $info = $this->get_option('block_description');
+    $block_title = $this->get_option('block_description');
+		$block_description = $this->get_option('block_layout_description');
 
-    if (empty($info)) {
+    if (empty($block_title)) {
       if ($this->display->display_title == $this->definition['title']) {
-        $info = t('View: @view', array('@view' => t($this->view->get_human_name())));
+        $block_title = t('View: @view', array('@view' => t($this->view->get_human_name())));
       }
       else {
-        $info = t('View: @view: @display', array('@view' => t($this->view->get_human_name()), '@display' => t($this->display->display_title)));
+        $block_title = t('View: @view: @display', array('@view' => t($this->view->get_human_name()), '@display' => t($this->display->display_title)));
```

Comment:
This entire if/else is pointless. We are repeating the entire thing just to append the display title.